### PR TITLE
Removing possibility to render map previews using cdb.Image

### LINKED ISSUE #4670: migrate previews to preview api
Remove all the cartodb.js instanciation thing and use the static url for the visualization:

https://github.com/CartoDB/cartodb/blob/master/app/views/admin/visualizations/embed_map.html.erb#L5

### PR DESCRIPTION
**Pull request by xavijam**
We used to have two possibilities, render the map previews with:

- Vizjson: it was using the cdb.Image we introduced in CartoDB.js (old method).
- Vizid: using the Static API we have in the editor (newish method).

Due to the fact that Preview API is faster that the old method, we have removed that one and we only offer de new one.

Places changed:

- Dashboard maps list.
- Delete items dialog.
- Public organization dashboard.
- Public user feed.
- Explore page.
- Public user maps.
- Public user datasets.
- Public map.
- Public dataset.
- Pecan?: no, it uses ```cdb.Image...```

Fixes #4670.
REVIEWER: @javierarce

cc @matallo

### CODE REVIEW

**Comment by xavijam** [comment]
Available on RUIs in a while.

File: lib/assets/javascripts/cartodb/explore/entry.js [hunk 1]
```diff
@@ -13,12 +13,9 @@ var ExploreModel = require('./model');
  */
 $(function() {
   cdb.init(function() {
-
     cdb.templates.namespace = 'cartodb/';
+    cdb.config.set(window.config);
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/explore/entry.js]: exposing all the configuration in the cdb.config model, where we have ```maps_api_template```, for example.

File: lib/assets/javascripts/cartodb/explore/view.js [hunk 2]
```diff
@@ -217,7 +217,7 @@ module.exports = Feed.extend({
         username: username,
         visId: visId,
         className: className,
-        account_host: cdb.config.get('account_host')
+        mapsApiHost: cdb.config.getMapsApiHost()
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/explore/view.js]: It should use the same approach we take in the whole editor, despite of the fact that explore page is a bit special. But if we need to check that works in local, we should "create" the same environment with a common-data user in development.

**Comment by javierarce** [comment]
### Checked
- [ ] Dashboard maps list.
- [ ] Delete items dialog.
- [ ] Public organization dashboard.
- [ ] Public user feed.
- [ ] Explore page.
- [ ] Public user maps.
- [ ] Public user datasets.
- [ ] Public map.
- [ ] Public dataset.
- [ ] Pecan

**Comment by xavijam** [comment]
Fixed problem when a map is shared in the organization and the map preview was not working due to it was not taking the map owner.

**Comment by javierarce** [comment]
:+1: code and manual tests :are :ok:

**Comment by xavijam** [comment]
Waiting the green light to deploy :dancer:, thanks @javierarce.

**Comment by xavijam** [comment]
Merged with master due to package conflict, on frontend green, :rocket:.

**Pull request merged**